concordium-cis2/src/lib.rs

```diff
@@ -31,6 +31,11 @@
 //! When `u256_amount` feature is enabled the type [`TokenAmountU256`] is defined
 //! and implements the [`IsTokenAmount`] interface.
 #![cfg_attr(not(feature = "std"), no_std)]
+
+mod cis2_client;
+pub use cis2_client::Cis2Client;
+pub use cis2_client::Cis2ClientError;
```

Comment:
@abizjak : just to highlight, also had to export `Cis2ClientError`.

Reply:
Good catch. Makes sense.

Reply:
You probably need to reformat though ;)